Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -738,42 +751,77 @@ final class EntityVisitor: SyntaxVisitor {
     }
 
     override func visit(_ node: ImportDeclSyntax) -> SyntaxVisitorContinueKind {
-        if let ret = node.path.firstToken(viewMode: .sourceAccurate)?.text {
-            let desc = node.importKeyword.text + " " + ret
-            imports["", default: []].append(desc)
-        }
+        imports.append(
+            .init(line: node.trimmedDescription)
+        )
         return .skipChildren
     }
 
     override func visit(_ node: IfConfigDeclSyntax) -> SyntaxVisitorContinueKind {
-        for cl in node.clauses {
-            let macroName: String
+        let blockId = Self.ifConfigDeclCount
```

Comment:
I think you need to use file names to stabilize the order. Alternatively, you might get some useful value from Swift Syntax's `Syntax`